crates/re_sdk/src/recording_stream.rs

```diff
@@ -1528,10 +1537,54 @@ impl RecordingStream {
         ThreadInfo::set_thread_time(
             &this.info.store_id,
             Timeline::new(timeline, TimeType::Time),
-            ns.into().map(|ns| Time::from_ns_since_epoch(ns).into()),
+            Time::from_ns_since_epoch(ns.into()).into(),
         );
     }
 
+    /// Clears out the current time of the recording for the specified timeline, for the
+    /// current calling thread.
+    ///
+    /// For example: `rec.disable_timeline("frame")`, `rec.disable_timeline("sim_time")`.
+    ///
+    /// See also:
+    /// - [`Self::set_timepoint`]
+    /// - [`Self::set_time_sequence`]
+    /// - [`Self::set_time_seconds`]
+    /// - [`Self::set_time_nanos`]
+    /// - [`Self::reset_time`]
+    pub fn disable_timeline(&self, timeline: impl Into<TimelineName>) {
+        let Some(this) = &*self.inner else {
+            re_log::warn_once!("Recording disabled - call to disable_timeline() ignored");
+            return;
+        };
+
+        let timeline = timeline.into();
+        ThreadInfo::unset_thread_time(&this.info.store_id, Timeline::new_sequence(timeline));
+        ThreadInfo::unset_thread_time(&this.info.store_id, Timeline::new_temporal(timeline));
+    }
+
+    /// Clears out the current time of the recording for the specified timeline of a specific
+    /// kind (see [`TimeType`]), for the current calling thread.
+    ///
+    /// For example: `rec.disable_timeline(Timeline::new_sequence("frame"))`.
+    ///
+    /// See also:
+    /// - [`Self::set_timepoint`]
+    /// - [`Self::set_time_sequence`]
+    /// - [`Self::set_time_seconds`]
+    /// - [`Self::set_time_nanos`]
+    /// - [`Self::disable_timeline`]
+    /// - [`Self::reset_time`]
+    #[doc(hidden)]
+    pub fn disable_timeline_specific(&self, timeline: impl Into<Timeline>) {
```

Comment:
remove or make non-`pub`, since it is will just confuse users

```suggestion
   fn disable_timeline_specific(&self, timeline: impl Into<Timeline>) {
```